src/index.js

```diff
@@ -0,0 +1,94 @@
+import 'lodash';
+import './style.css';
+import elementChanges from './check.js';
+
+let ToDoArray = JSON.parse(localStorage.getItem('ToDoArray') || '[]');
+let title = JSON.parse(localStorage.getItem('title'));
+if (!ToDoArray.length) {
+  document.querySelector('.Activity-list').classList.add('hidden');
+} else {
+  document.querySelector('.Activity-list').classList.remove('hidden');
+}
+document.querySelector('#whatToDo').value = title;
+
+function CreateOb({ description, index }) {
+  this.description = description;
+  this.completed = false;
+  this.index = index;
+}
+
+function displayOnScreen(element) {
+  const ul = document.querySelector('.form_');
+  const li = document.createElement('li');
+  li.classList.add('toDo');
+  const checkbox = document.createElement('input');
+  const br = document.createElement('br');
+  const image = document.createElement('img');
+  image.setAttribute('src', 'icons/delete.png');
```

Comment:
Your image isn't displaying well kindly fix it and request another code review.
<img width="1552" alt="Screen Shot 2021-12-02 at 17 17 01" src="https://user-images.githubusercontent.com/22258274/144518171-fc54d347-1263-4946-bdea-51a4333fa3c1.png">